pkgs/development/libraries/pango/default.nix

```
@@ -22,7 +22,7 @@ in stdenv.mkDerivation rec {
   outputs = [ "bin" "dev" "out" ] ++ optional (!stdenv.isDarwin) "devdoc";
 
   nativeBuildInputs = [
-    meson ninja
+    meson ninja glib
```

Possibly glib-mkenums?


```suggestion
    meson ninja
    glib # for glib-mkenum
```

---

yeah